README.md

```diff
@@ -5,6 +5,7 @@
 [![CI](https://img.shields.io/github/actions/workflow/status/meshtastic/python/ci.yml?branch=master&label=actions&logo=github&color=yellow)](https://github.com/meshtastic/python/actions/workflows/ci.yml)
 [![CLA assistant](https://cla-assistant.io/readme/badge/meshtastic/python)](https://cla-assistant.io/meshtastic/python)
 [![Fiscal Contributors](https://opencollective.com/meshtastic/tiers/badge.svg?label=Fiscal%20Contributors&color=deeppink)](https://opencollective.com/meshtastic/)
+![GPL-3.0](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)
```

Comment:
It looks like this is showing an LGPL badge, but presumably it should be https://img.shields.io/badge/License-GPL%20v3-blue.svg instead?

Reply:
Good spot, fixed.